scipy/_build_utils/src/wrap_g77_abi_ilp64.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * This file is an analog of wrap_g77_abi.c, only for the ILP64 BLAS build.
+ * See `wrap_g77_abi.c` for the purpose and scope.
+ *
+ * The only difference between this and wrap_g77_abi.c is that here we
+ * export the wrappers mangled with BLAS_FUNC (i.e. with BLAS_PREFIX) instead
+ * of mangling with F_FUNC from fortran_defs.h.
+ * This mirrors the usage in scipy/linalg/fblas_64.pyf.src
```

Comment:
Did you consider avoiding the duplication by, for example, defining a new `WRAP_BLAS_FUNC` which is an alias to either `BLAS_FUNC` or `F_FUNC` depending on whether `HAVE_BLAS_ILP64` is defined?